5section_methods_plus_calc.rb

```diff
@@ -0,0 +1,57 @@
+def calculator(n1, n2, op)
+    case op
+    when 1
+        return n1+n2
+    when 2
+        return n1-n2
+    when 3
+        return n1*n2
+    when 4 
+        return n1.to_f/n2
+    else
+        return "Please enter a valid operation"
+    end
+end
+
+def are_odd(n1, n2)
+    # p "in odd"
```

Comment:
For the indentation use 2 spaces: https://github.com/rubocop/ruby-style-guide#spaces-indentation